pkg/daemon/daemon.go

```diff
@@ -248,7 +248,8 @@ func (d *daemon) AddPeriodicUpdate() {
 				pod.Annotations[v1.NetworkAttachmentAnnot] = string(netAnnotations)
 			}
 
-			guidList = append(guidList, guidAddr)
+			// used GUID as net.HardwareAddress to use it in sm plugin which receive n[]et.HardwareAddress as parameter
```

Comment:
the comment is not clear also what is the []
return net.HardwareAddress has other modules such as sm plugin still use it.
We should fix this in another PR

Reply:
Apart from `guid` module, the `net.HardwareAddress` is used in `daemon`, `ib-utils`, and `sm plugins`.

Reply:
@moshe010 updated the comment